src/gui/tray/ActivityItem.qml

```diff
@@ -1,264 +1,123 @@
-import QtQml 2.12
-import QtQuick 2.9
-import QtQuick.Controls 2.2
+import QtQml 2.15
+import QtQuick 2.15
+import QtQuick.Controls 2.3
 import QtQuick.Layouts 1.2
 import Style 1.0
-import com.nextcloud.desktopclient 1.0
 
 MouseArea {
```

Comment:
A possible amelioration for later would be to port this to QtQuick.Controls2.ItemDelegate. It would simplify the code a bit :)

Reply:
@CarlSchwan we might do it next time during refactoring